src/checkpoint/checkpoint_txn.c

```diff
@@ -1063,9 +1064,10 @@ __txn_checkpoint(WT_SESSION_IMPL *session, const char *cfg[])
     uint64_t ckpt_tree_duration_usecs, fsync_duration_usecs, generation, hs_ckpt_duration_usecs;
     uint64_t time_start_ckpt_tree, time_start_fsync, time_start_hs, time_stop_ckpt_tree,
       time_stop_fsync, time_stop_hs;
-    u_int i;
+    u_int i, ckpt_total_steps, ckpt_relative_crash_step;
+    int ckpt_crash_random;
```

Comment:
Nitpick: Maybe rename this to `ckpt_crash_point` to better match the config parameter name `checkpoint_crash_point`?

Reply:
Fixed in 9d64d04ec275d9d111c19e36ac8584074dda0211